test/e2e/e2ethanos/services.go

```diff
@@ -242,6 +243,50 @@ func NewReceiver(sharedDir string, networkName string, name string, replicationF
 	return receiver, nil
 }
 
+func NewReceiverWithDistributorMode(sharedDir string, networkName string, name string, replicationFactor int, hashring ...receive.HashringConfig) (*Service, error) {
```

Comment:
Naming nit: there is no concrete thing as "distribution" mode, but we can talk concretely about whether the receiver supports ingesting to a TSDB or not. I would propose calling this `NewReceiverWithoutTSDB`

Reply:
But not a hard blocker here

Reply:
Lets add this to the suggestions, should be consistent with the nomenclature :)

Reply:
Done